index.html

```diff
@@ -1,25 +0,0 @@
----
-layout: default
----
-
-<div class="home">
-
-  <h1 class="page-heading">Posts</h1>
-  
-  <a class='githublink' href="https://github.com/resque/resque">Source code and documentation on github</a>
```

Comment:
@varyonic we should keep the github link on the page.

Reply:
Added back 'View on Github', I'll leave styling to someone else.

I'm a little reluctant to pull in the plugins page.  I understand the wiki does not get indexed by Google but it has minimal barriers to contribution so anyone in the community can register their forks and add notes on older gems (eg. @pboling).  I do think the docsite should link to other gems adopted by the organization, ie. resque-scheduler (will resque-loner be revived?)  Which other plugins might be adopted?

Reply:
> but it has minimal barriers to contribution so anyone in the community can register their forks and add notes on older gems 

Yeah, this is a downside. I wonder if there's some GH tooling out there to sync GH wiki pages to a public website or something? Would be something useful to look into.

What gems/projects do this well? None really come to mind for me.

Reply:
Will `resque-loner` be revived?  Unlikely, I think.  However, the lineage is alive (and yet withering, as I don't currently use Resque).
My `resque-unique_in_queue` was a rewrite of `resque-solo`, which was a rewrite of `resque-loner`. :)

Reply:
Right. I'm currently switching from resque-loner to resque-unique-at-runtime, but given that resque-loner has already been adopted by the resque organization perhaps PRs could be accepted for a v2.0?  If not then may as well archive it.